### PR TITLE
fix(curriculum): use two-space tab in HTML and JS code of RPG project

### PR DESCRIPTION
**Pull request by huyenltnguyen**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

This PR contains the following changes to the RPG project:
- Fix the indentation of the `<head>` and `<body>` elements in the HTML file
- Fix the indentation width of the HTML file: it's using 4-space tab while we have been using 2-space tab for HTML
- Fix the indentation width of the `locations` array in the JS file: it's using 4-space tab while we have been using 2-space tab for JS
- Remove an empty line from the `locations` array in step 159 (which explains the one-line difference between the code additions and deletions)
  - Link to the challenge: https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-basic-javascript-by-building-a-role-playing-game/step-159
  - File name: `62aa252c3b1073415ba2b898.md`

**Note:** I manually made the changes so it's very likely that I missed a couple spots. If there is a way to automate the process, don't tell me, I don't want to know 😢 

## Testing

I randomly picked some steps and checked if the code is displayed properly. I also ran `pnpm run test:curriculum` and all tests passed.

### CODE REVIEW

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a1166ed9a56d439c0770e7.md [hunk 1]
```diff
@@ -49,22 +49,22 @@ assert(document.querySelectorAll('#stats > .stat')?.[2]?.innerText === 'Gold: 50
 ```html
 <!DOCTYPE html>
 <html lang="en">
-<head>
+  <head>
     <meta charset="utf-8">
     <link rel="stylesheet" href="./styles.css">
     <title>RPG - Dragon Repeller</title>
 </head>
```
Comment (gikf) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a1166ed9a56d439c0770e7.md:56]:
```suggestion
  </head>
```

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a94114ce0b8918b487390f.md [hunk 2]
```diff
@@ -54,33 +54,33 @@ assert.equal(locations[6].text, 'You defeat the dragon! YOU WIN THE GAME! 🎉')
 ```html
 <!DOCTYPE html>
 <html lang="en">
-<head>
+  <head>
     <meta charset="utf-8">
     <link rel="stylesheet" href="./styles.css">
     <title>RPG - Dragon Repeller</title>
 </head>
```
Comment (gikf) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a94114ce0b8918b487390f.md:61]:
```suggestion
  </head>
```

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62b46e3a8d4be31be5af793d.md [hunk 3]
```diff
@@ -40,31 +40,31 @@ assert.equal(script.parentElement.tagName, "BODY");
...
+      <div id="monsterStats">
+        <span class="stat">Monster Name: <strong><span id="monsterName"></span></strong></span>
+        <span class="stat">Health: <strong><span id="monsterHealth"></span></strong></span>
+      </div>
+      <div id="text">
+        Welcome to Dragon Repeller. You must defeat the dragon that is preventing people from leaving the town. You are in the town square. Where do you want to go? Use the buttons above.
+      </div>
     </div>
 
 </body>
```
Comment (gikf) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62b46e3a8d4be31be5af793d.md:70]:
```suggestion
  </body>
```

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8eff21c0b0f6ebe5b8e38.md
Comment (gikf) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8eff21c0b0f6ebe5b8e38.md:1]: There's single space on blank line here - line 15.

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8aa98a8289d0a698eee1d.md [hunk 4]
```diff
@@ -143,9 +143,9 @@ const monsterName = document.querySelector("#monsterName");
 const monsterHealthText = document.querySelector("#monsterHealth");
 --fcc-editable-region--
 const locations = [
-    {
+  {
```
Comment (gikf) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8aa98a8289d0a698eee1d.md:147]:
```suggestion
    
```

**Review by gikf — COMMENTED** [review]
I used search with regex to find these edge cases :eyes:. So I also can confirm that's the only occurrences of them.

**Comment by gikf** [comment]
I used all the weird ones, when searching in VS Code in the project folder, among others:
```
^<[^h!]
^</[^h!]
^\s+$
^\s{3}<
^\s{5}<
```

**Review by gikf — APPROVED** [review]

**Review by Sembauke — APPROVED** [review]

**Pull request merged**